src/ts/background/index.ts

```diff
@@ -1,22 +1,11 @@
-import {createStore} from 'redux';
-import {wrapStore} from 'webext-redux';
-import {configureApp} from './AppConfig';
-import reducers, {loadState} from './store';
-import {browser} from 'webextension-polyfill-ts';
-
-const preloadedState = loadState();
-const store = createStore(reducers, preloadedState);
-
-configureApp(store);
-wrapStore(store);
-
-browser.commands.onCommand.addListener((command) => {
```

Comment:
was removing of this one intentional? (as this breaks browser-based shortucs, but the commands are still defined in  manifest  and this CR does not define shortcuts for all actions)

Reply:
It was intentional, but ultimately I didn't want to modify your srs stuff and forgot about that

Reply:
I mean if we go this way we go all in :p